### PR TITLE
 Contact form and Footer: Mobile Version

### PR DESCRIPTION
**Pull request by smunozmo**
# Template 5
[Figma Template #5 ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1095)
 
## Updates:

* Fixed form and text alignment. 
* Created Contact form and Footer sections.
* Added HTML to Contact form.
* Added CSS style to Contact form and Footer.

No linters errors detected. ✅
Used flex-box for layout. ✅

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,5 +178,44 @@ <h2>Skills</h2>
...
+            <img src="assets/img/ic_link_teal.svg" alt="link-teal" width="22">
+        </div>            
+        <div class="social">
+            <img src="assets/img/ic_stackoverflow.svg" alt="">
+            <img src="assets/img/ic_github.svg" alt="">
+            <img src="assets/img/ic_twitter.svg" alt="">
+            <img src="assets/img/ic_linkedin.svg" alt="">
+            <img src="assets/img/ic_medium.svg" alt="">
+            <img src="assets/img/ic_angellist.svg" alt="">
+        </div>
```
Comment (Davidosky007) [inline index.html]: I suggest you wrap this section in a `ul`  tag and each `img` tag inside a `li` tag this is best practice read [This](https://stackoverflow.com/questions/13416866/why-should-i-use-ul-li-in-my-html) for further information

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi, @Kambuchi @smunozmo, 
# you shouldn't close a previous pull request but if you did that by mistake please remember to share a link to it, so the reviewer doesn't need to look for it. your project would be marked as invalid in the future if repeated intentionally 🙏 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
# _Check the comments under the review make the necessary changes and submit for review using this same PR 👍 ._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Davidosky007) [inline index.html]: I suggest you stick to the design templates in Figma as much as possible, this is how your design looks
Screenshot (32)

This is the Figma
Screenshot (31)
 design template compare and make the necessary changes

File: css/style.css [hunk 2]
```diff
@@ -254,3 +254,142 @@ h2 {
...
+  flex-direction: column;
+}
+
+.contact-form h1 {
+  padding: 106px 0 0 16px;
+  font-family: Playfair Display, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
```
Comment (sinansevgi) [inline css/style.css:276]: [Optional]
- [ ] This causes a vertical scrollbar to appear on the title. Please remove this

File: index.html [hunk 3]
```diff
@@ -180,5 +180,56 @@ <h2>Skills</h2>
...
+                   <span>E-mail:</span>
+                   <input type="email" id="mail" name="user_email" placeholder="mister@bean.com" required>
+                 </div>
+                 <div class="float-textarea">
+                   <span></span>
+                   <textarea id="msg" name="user_message" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                 </div>
+                </div>
+                <div class="btn">
+                    <button>Start collaboration</button>
```
Comment (sinansevgi) [inline index.html:204]: [Optional]

- [ ] Button doesn't have a proper margin. Please check the template and put the proper margin on this button.

File: css/style.css [hunk 4]
```diff
@@ -254,3 +254,142 @@ h2 {
...
+  position: relative;
+  width: 288px;
+}
+
+/* Contact form end */
+
+footer {
+  background-color: #1a2236;
+  width: 320px;
+  height: 150px;
```
Comment (sinansevgi) [inline css/style.css:359]: [Optional]
- [ ] This height is too small for this section. Content overflows the container. Please avoid giving static height and width to elements. If you want to create space in your container. You should use margin and padding.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 
------

Hi @smunozmo , 

Good job so far!

You are using the correct font now. :+1: 

I can see that your design is still a little bit different from the template as the previous TSE reported. But since this is your last review and the differences are quite little, I will approve this pull request and leave my suggestions as **[Optional]**. But I strongly recommend you to work on these suggestions. 

You can merge this pull request and continue with your journey :wave: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**